docs/user-manual/en/intercepting-operations.md

```diff
@@ -9,7 +9,9 @@ makes interceptors powerful, but also potentially dangerous.
 
 ## Implementing The Interceptors
 
-An interceptor must implement the `Interceptor interface`:
+All interceptors are protocol specific.
+
+An interceptor for the core protocol must implement the interface `Interceptor`:
```

Comment:
I'd change that to 

    implement the `Interceptor` interface

sounds more like standard Java-speak to me that way.